src/main/java/com/kodilla/checkers/gamelogic/GameTreeNode.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.checkers.gamelogic;
+
+import com.kodilla.checkers.gameio.BoardIO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameTreeNode {
```

Comment:
Ta klasa jest bardzo abstrakcyjna. Jeśli używasz jej do wyznaczenia najdłuższej ścieżki dla komputera, to warto zamknąć to w jakimś pakiecie i nadać temu konkretniejszą nazwę.